pkg/kubelet/kubelet.go

```diff
@@ -222,7 +222,7 @@ func getContainerEtcHostsPath() string {
 
 // SyncHandler is an interface implemented by Kubelet, for testability
 type SyncHandler interface {
-	HandlePodAdditions(pods []*v1.Pod)
+	HandlePodAdditions(ctx context.Context, pods []*v1.Pod)
```

Comment:
Sorta thinking it makes sense to plumb the context into HandlePod{Updates,Removes,Reconcile} and friends. Thoughts?

Reply:
yeah, looking at this file I immidiately felt the same. Just hate how big the PR will become and was thinking to take this first step